common/app/views/fragments/header.scala.html

```diff
@@ -58,7 +58,7 @@
                         <a class="top-bar__item hide-until-tablet js-subscribe js-acquisition-link"
```

Comment:
Just above here we have logic to say "Make a contribution" in the US but "Support The Guardian" elsewhere - could be worth checking with @ionamckendrick, Amanda etc if we still want that distinction

Reply:
although to be fair - that's the header not the side menu!

Reply:
I would try to reduce the scope of this PR. I [created a card](https://trello.com/c/nlDNNkYd/308-should-we-make-the-make-a-contribution-button-in-the-header-the-same-acroos-all-the-regions) to attack that in the future.